core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -274,12 +274,20 @@ protected void startJvmMonitorProcess() {
    * @param stopMessage empty string or the message that the master loses the leadership
    */
   protected void startServing(String startMessage, String stopMessage) {
+    startCommonServing(startMessage);
+    startLeaderServing(startMessage, stopMessage);
+  }
+
+  protected void startCommonServing(String startMessage) {
     MetricsSystem.startSinks(ServerConfiguration.get(PropertyKey.METRICS_CONF_FILE));
-    startServingRPCServer();
     LOG.info("Alluxio master web server version {} starting{}. webAddress={}",
         RuntimeConstants.VERSION, startMessage, mWebBindAddress);
```

Comment:
Move this log into `startServingWebServer()`

Reply:
Done